editor/src/components/editor/image-insert.ts

```diff
@@ -10,10 +10,16 @@ async function fileUploadAction(file: File, targetPath: string): Promise<EditorA
   return fileResultUploadAction(fileResult, targetPath)
 }
 
-export function fileResultUploadAction(fileResult: FileResult, targetPath: string): EditorAction {
+export function fileResultUploadAction(
+  fileResult: FileResult,
+  targetPath: string,
+  replace: boolean = false,
```

Comment:
How much work is it to make this param non-optional? If it's not a lot, please can you do that?

Reply:
fixed